willsCheatMenu/src/index.ts

```diff
@@ -140,6 +140,17 @@ if (localStorage.getItem("level")) {
 	eval(`_.player.getLevel = () => {return ${localStorage.getItem("level")}}`)
 }
 
+if (localStorage.getItem("pets")) {
+	let pets = JSON.parse(localStorage.getItem("pets"))
+	pets.forEach(element => {
+		_.player.kennel.petTeam.forEach(pet => {
+			if (pet.getID() == element[0]) {
+				eval(`_.player.kennel.petTeam[${element[0]+1}].getLevel = () => {return ${element[1]}}`)
```

Comment:
I don't know how you managed it but this is one of the worst lines of code I've ever seen